packages/sdk-py/src/agent_relay/communicate/transport.py

```
@@ -260,128 +218,114 @@ async def check_inbox(self) -> list[Message]:
+        event = self._ws_open
+        if event is not None:
+            event.set()
+
+    def _on_ws_ping(self, _payload: dict[str, Any]) -> None:
+        ws = self._ws
+        if ws is None:
             return
+        # _send_json is the only send primitive WsClient exposes; schedule a
+        # pong on the running loop.
+        with suppress(Exception):
+            asyncio.ensure_future(ws._send_json({"type": "pong"}))
```

Ruff (0.15.18)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)